src/openziti/__main__.py

```diff
@@ -15,7 +15,7 @@
 from . import zitilib
 
 
-def do_enroll():
+def do_enroll(args):
     with open(args.jwt) as jwt_file:  # pylint: disable=unspecified-encoding
```

Comment:
would it be better to specify encoding instead of ignoring pylint?

Reply:
This one is really up to you. I added the pylint disable in a different PR because this looked intentional, as sometimes you want the platform dependent encoding to be used. As written, we open the jwt with with the following expanded form:

`open(args.jwt, mode='r', encoding=locale.getpreferredencoding(False), ...)`

You can read more about it to decide if that's what you intended here:
[open()](https://docs.python.org/3/library/functions.html#open)

In either case, it's not part of this PR.